src/main/java/duke/SaveFile.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SaveFile {
```

Comment:
Class can be named better. Save file gives the impression that this class only saves files and not read them

Reply:
Thank you, I have renamed the class to something more appropriate.